crates/proof-of-sql/tests/timestamp_integration_tests.rs

```diff
@@ -134,6 +134,35 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_basic_timestamp_inequality_query() {
+        let test_timestamps = vec![i64::MIN, -1, 0, 1, i64::MAX];
+
+        run_timestamp_query_test(
+            "SELECT * FROM table WHERE times < timestamp '1970-01-01T00:00:00Z';",
```

Comment:
We need filters in tests more complex than `column < literal`. Please at least try to compare two columns in one of the tests.